lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -45,10 +49,23 @@ func Init(portLayerAddr string) error {
 	portLayerServerAddr = portLayerAddr
 
 	imageCache = cache.NewImageCache()
-	// update the image cache at startup
-	if err := imageCache.Update(portLayerClient); err != nil {
-		return fmt.Errorf("Error refreshing image cache: %s", err)
-	}
+
+	// attempt to update the image cache at startup
+	log.Info("Refreshing image cache...")
+	go func() {
+		for i := 0; i < Retries; i++ {
+
+			// initial pause to wait for the portlayer to come up
+			time.Sleep(RetryTimeSeconds * time.Second)
+
+			if err := imageCache.Update(portLayerClient); err == nil {
+				log.Info("Image cache updated successfully")
+				return
+			}
+			log.Info("Failed to refresh image cache, retrying...")
+		}
+		log.Warn("Failed to refresh image cache. Is the portlayer server down?")
+	}()
 
 	return nil
```

Comment:
The portlayer has a ping request.  We can keep trying to check if the portlayer's rest server is up before bringing up the personality's rest server.